chrome.json

```diff
@@ -1 +1 @@
-{ "executablePath": "" }
\ No newline at end of file
+{"executablePath":"/usr/bin/google-chrome-stable"}
```

Comment:
I think you accidentally add this line. `executablePath` should be empty by default.